app/data/census_individual.json

```diff
@@ -1922,15 +1940,12 @@
                                                     "value": "Irish"
                                                 },
                                                 {
-                                                    "label": "None",
-                                                    "value": "None"
+                                                    "label": "Other",
+                                                    "value": "Other"
                                                 },
                                                 {
-                                                    "label": "Other (please specify)",
-                                                    "value": "Other",
-                                                    "other": {
-                                                        "label": "Please specify other passports held"
-                                                    }
+                                                    "label": "None",
+                                                    "value": "None"
```

Comment:
Okay, fair enough, just highlighting it may need to be tested to ensure it works correctly. 👍